morpheushelper/morpheushelper.py

```diff
@@ -1,360 +1,36 @@
-        description=GITHUB_DESCRIPTION,
-        colour=Colours.github,
-        url=GITHUB_LINK,
-    )
-    embed.set_author(name="GitHub", icon_url="https://github.com/fluidicon.png")
-    embed.set_thumbnail(url=AVATAR_URL)
-    await ctx.send(embed=embed)
-
-
-@bot.command(name="version")
-async def version(ctx: Context):
-    """
-    show version
-    """
-
-    embed = Embed(title=f"MorpheusHelper v{VERSION}", colour=Colours.version)
-    await ctx.send(embed=embed)
-
-
-@bot.command(name="info", aliases=["infos", "about"])
-async def info(ctx: Context):
-    """
-    show information about the bot
-    """
-
-    await send_long_embed(ctx, await build_info_embed(False))
-
-
-@bot.command(name="admininfo", aliases=["admininfos"])
-@Permission.admininfo.check
-async def admininfo(ctx: Context):
-    """
-    show information about the bot (admin view)
-    """
-
-    await send_long_embed(ctx, await build_info_embed(True))
-
-
-@bot.event
-async def on_error(*_, **__):
-    if sentry_dsn:
-        sentry_sdk.capture_exception()
-    else:
-        raise  # skipcq: PYL-E0704
-
-
-@bot.event
-async def on_command_error(ctx: Context, error: CommandError):
-    if isinstance(error, CommandNotFound) and ctx.guild is not None and ctx.prefix == await get_prefix():
-        messages = []
-    elif isinstance(error, UserInputError):
-        messages = await send_help(ctx, ctx.command)
-    else:
-        messages = [await ctx.send(embed=make_error(error))]
-
-    if ctx.author.id == 370876111992913922:
-        sudo_cache[ctx.channel] = ctx.message
-
-    add_to_error_cache(ctx.message, messages)
-
-
-@listener
-async def on_bot_ping(message: Message):
-    await message.channel.send(embed=await build_info_embed(False))
-
-
-cog_blacklist = set(map(str.lower, os.getenv("DISABLED_COGS", "").split(",")))
-disabled_cogs = []
-enabled_cogs = []
-for cog_class in COGS:
-    if cog_class.__name__.lower() in cog_blacklist or (cog_class is AdventOfCodeCog and not AOCConfig.load()):
-        disabled_cogs.append(cog_class.__name__)
-        continue
-
-    enabled_cogs.append(cog_class)
+    "\033[0m",
+)
 
-register_cogs(bot, *enabled_cogs)
+logger.info(f"Starting {Config.NAME} v{Config.VERSION} ({Config.REPO_LINK})\n")
 
-if bot.cogs:
-    root_logger.info("\033[1m\033[32m%s Cog%s enabled:\033[0m", len(bot.cogs), "s" if len(bot.cogs) > 1 else "")
-    for cog in bot.cogs.values():
-        commands = ", ".join(cmd.name for cmd in cog.get_commands())
-        root_logger.info(" + %s %s", cog.__class__.__name__, str(commands) if commands else "")
-if disabled_cogs:
-    root_logger.info(
-        "\033[1m\033[31m%s Cog%s disabled:\033[0m", len(disabled_cogs), "s" if len(disabled_cogs) > 1 else ""
-    )
-    for name in disabled_cogs:
-        root_logger.info(" - %s", name)
+if SENTRY_DSN:
+    logger.debug("initializing sentry")
+    setup_sentry(SENTRY_DSN, Config.NAME, Config.VERSION)
 
-bot.run(os.environ["TOKEN"])
+__import__("bot").run()
```

Comment:
How about if u make the import at the top level and execute it at the end

Reply:
this is not possible because the config needs to be loaded before `bot` is imported